server/plugin/api.go

```diff
@@ -496,6 +499,22 @@ func (p *Plugin) handleSubscriptionNotifications(w http.ResponseWriter, r *http.
 			sourceBranchName = strings.Split(body.Resource.PullRequest.SourceRefName, "/")[2]
 		}
 
+		// Convert map to json string
+		jsonStr, err := json.Marshal(body.Resource.Comment)
```

Comment:
i think json.Marshal returns a byte array and not a string so this variable name could be misleading